src/errors/APIError.ts

```diff
@@ -221,6 +221,7 @@ export enum ResponseErrorCode {
 
     /* Gateway simulation */
     PlatformCredentialsDisabled = "PLATFORM_CREDENTIALS_DISABLED",
+    TaggedPlatformCredentialsDisabled = "TAGGED_PLATFORM_CREDENTIAL_DISABLED",
```

Comment:
perhaps make this consistent? minor typo on the `CREDENTIAL` > `CREDENTIALS`

Reply:
Ah thanks 🙂 For now the error is with the typo. I will notify the API about it and update it in a later PR. 🙂

Reply:
Also added another message I happened to see.

Reply:
Hum... After thinking about it. Fixed it in here. This wont break and it will be ready for the fixed error. Thanks for catching it! 🙂